### PR TITLE
Feature email functionality

### PR DESCRIPTION
**Pull request by purisaurabh**

### CODE REVIEW

File: package-lock.json
Comment (Payal13) [inline package-lock.json:1]: please check what are the changes and why this got added?

File: src/components/Builder/BasicInfo/index.jsx [hunk 1]
```diff
@@ -145,8 +152,16 @@ const BasicInfo = ({ profileData }) => {
               {
                 pattern: WHOLE_NO_VALIDATOR,
                 message: "Experience must be a whole number",
-              }
-            ]}           
+              },
+              {
```
Comment (Payal13) [inline src/components/Builder/BasicInfo/index.jsx]: remove this validator, we dont want any validation on this field

**Review by Payal13 — APPROVED** [review]

File: package-lock.json
Comment (Abhishek-Jain-1925) [reply package-lock.json:1]: Done!

File: src/components/Builder/BasicInfo/index.jsx (hunk 1 above)
Comment (Abhishek-Jain-1925) [reply src/components/Builder/BasicInfo/index.jsx]: Done!

**Review by Payal13 — APPROVED** [review]
We can refactore this PR more, but merging because of time constraints. 
@Abhishek-Jain-1925 @purisaurabh Please refactore and reuse the code as much as possible.

**Pull request merged**